cmd/dowload_test.go

```diff
@@ -59,34 +65,10 @@ func TestRunDownloadWithTags(t *testing.T) {
 	cmd.SetArgs([]string{"-f", testfilepath, "download", "--tag", "tag", "--dir", outputDir})
 	err := cmd.Execute()
 	assert.Nil(t, err)
-	for _, file := range []string{"test.html"} {
-		test.AssertFileContains(t, fmt.Sprintf("%s/%s", outputDir, file), content)
-	}
-}
-
-func TestRunDownloadWithoutTags(t *testing.T) {
```

Comment:
Is there a reason to delete this test?

Reply:
In the interest of keeping the change minimal you should just keep the existing tests and add *only* the tests that are necessary to test your change, no remove existing existing tests.

Reply:
I took the liberty of un-resolving this comment: let's go ahead and put the test back.  As @rabadin noted: let's keep all existing tests and only *add* new tests that are testing your change.